README.adoc

```diff
@@ -255,8 +255,21 @@ aws dynamodb create-table --table-name delta_rs_lock_table \
         ReadCapacityUnits=10,WriteCapacityUnits=10
 ```
 
+== Schema Support
+This application has support for both avro and json format via command line arguments. If no format argument is provided, the default behavior is to use json.
+The table below indicates what will happen with respect to the provided arguments.
+| Argument      | Value |  Result |
```

Comment:
```suggestion

|===
| Argument      | Value |  Result |
```

This needs some fencing to be an [asciidoc table](https://docs.asciidoctor.org/asciidoc/latest/tables/build-a-basic-table/)